cmd/cloud-controller-manager/app/options/options.go

```diff
@@ -269,6 +269,5 @@ func createRecorder(kubeClient clientset.Interface, userAgent string) record.Eve
 	eventBroadcaster := record.NewBroadcaster()
 	eventBroadcaster.StartLogging(klog.Infof)
 	eventBroadcaster.StartRecordingToSink(&v1core.EventSinkImpl{Interface: kubeClient.CoreV1().Events("")})
-	// TODO: remove dependence on the legacyscheme
-	return eventBroadcaster.NewRecorder(legacyscheme.Scheme, v1.EventSource{Component: userAgent})
+	return eventBroadcaster.NewRecorder(runtime.NewScheme(), v1.EventSource{Component: userAgent})
```

Comment:
I'm not super familiar with the event broadcaster, but I think we need to pass in a `Scheme` that has the GVK for events installed.

I think using the client-go scheme would work (`k8s.io/client-go/kubernetes/scheme`)? @cheftako can you confirm^ ?

Reply:
We definitely need a scheme with the GVKs for any resources we use (which here is Events). I think client-go's scheme makes the most sense.